Traceback (most recent call last):
  File "/afs/cern.ch/user/l/lfavilla/CMSSW_12_6_0/src/PhysicsTools/NanoAODTools/python/postprocessing/my_analysis/my_framework/Utilities/FillHistos.py", line 238, in <module>
    Tops_Over_Thr = TopsOverThr(trees=trees, pt_flag=pt_flag, truth=truth, thr=thr)
  File "/afs/cern.ch/user/l/lfavilla/CMSSW_12_6_0/src/PhysicsTools/NanoAODTools/python/postprocessing/my_analysis/my_framework/Utilities/FillHistos.py", line 109, in TopsOverThr
    TopsOverThr.append(ak.flatten(tree[f"Top{pt_flag}Pt"][(tree[f"Top{pt_flag}Pt"]["truth"]==truth) * (tree[f"Top{pt_flag}Pt"]["scoreDNN"]>=thr)]))
  File "/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/py3-numpy/1.22.4-62e28bace1c65ef46c4a0988f2d65108/lib/python3.9/site-packages/numpy/lib/mixins.py", line 21, in func
    return ufunc(self, other)
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/awkward/highlevel.py", line 1411, in __array_ufunc__
    return ak._connect._numpy.array_ufunc(ufunc, method, inputs, kwargs)
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/awkward/_connect/_numpy.py", line 250, in array_ufunc
    out = ak._util.broadcast_and_apply(
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/awkward/_util.py", line 1172, in broadcast_and_apply
    out = apply(broadcast_pack(inputs, isscalar), 0, user)
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/awkward/_util.py", line 925, in apply
    outcontent = apply(nextinputs, depth + 1, user)
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/awkward/_util.py", line 1028, in apply
    outcontent = apply(nextinputs, depth + 1, user)
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/awkward/_util.py", line 750, in apply
    nextinputs.append(x.array)
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/awkward/operations/convert.py", line 4745, in _form_to_layout
    container[key_format(form_key=fk, attribute="data", partition=partnum)]
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/coffea/nanoevents/mapping/base.py", line 90, in __getitem__
    stack.append(self.extract_column(handle, start, stop))
  File "/afs/cern.ch/user/l/lfavilla/.local/lib/python3.9/site-packages/coffea/nanoevents/mapping/uproot.py", line 149, in extract_column
    return columnhandle.array(
  File "/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/py3-uproot/4.3.5-d4c15180c692dbe2161e59bf1541b5d3/lib/python3.9/site-packages/uproot/behaviors/TBranch.py", line 2208, in array
    _ranges_or_baskets_to_arrays(
  File "/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/py3-uproot/4.3.5-d4c15180c692dbe2161e59bf1541b5d3/lib/python3.9/site-packages/uproot/behaviors/TBranch.py", line 3405, in _ranges_or_baskets_to_arrays
    hasbranches._file.source.chunks(ranges, notifications=notifications)
  File "/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/py3-uproot/4.3.5-d4c15180c692dbe2161e59bf1541b5d3/lib/python3.9/site-packages/uproot/source/xrootd.py", line 371, in chunks
    status = self._resource.file.vector_read(
  File "/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/xrootd/5.5.1-7737cc2c22e178882ee7e369b85a17e2/lib/python3.9/site-packages/XRootD/client/file.py", line 236, in vector_read
    return XRootDStatus(self.__file.vector_read(chunks, timeout, callback))
ValueError: I/O operation on closed file
rm: cannot remove '../../../runner_QCD_HT300to500_2018.sh': No such file or directory
